suite-native/feature-flags/src/useFeatureFlag.e2e.ts

```diff
@@ -0,0 +1,6 @@
+import { launchArguments } from '@suite-native/config';
+
+import { FeatureFlag } from './featureFlagsSlice';
+
+export const useFeatureFlag = (featureFlag: FeatureFlag): boolean =>
```

Comment:
What about getting the default value from the state and only allowing it to be overridden via a launch argument? This way, we wouldn’t have to define every new feature flag in yet another place. It would be nice to keep a single source of truth and only allow overrides where needed.

Reply:
https://github.com/trezor/trezor-suite/pull/20277/commits/86de8df0a67a34c016000b601dfaec865fd35465